docs/Demo.md

```diff
@@ -134,49 +229,203 @@ curl $(oc get route -n pelorus deploytime-exporter -o=template='http://{{.spec.h
 
 ### See the change to your todolist application
 You can now also check on your todolist application and see the updated text change "Add a todo"
+
 ![todolist-fixed](img/todolist_fixed.png)
 
 
 ## Troubleshooting
 
 If exporters are not functioning or deployed, no data will show up in the dashboard. It will look like the following:
+
 ![No-Data](img/pelorus-dashboard-no-data.png)
 
 * Please check the logs of exporter pod.
 
 An "idle" state could resemble:
+
 ![Idle-Data](img/pelorus-dashboard-idle-data.png)
 
 
-## Automated Demo
+# Mean Time to Restore and Change Failure Rate
+
+![dora_fail_recover](img/dora_metrics2.png)
+
+### Assumptions
+- Github issues are enabled in https://github.com/your_org/mig-demo-apps/settings
+
+### Workflow - Failure and Resolution
+- Create and resolve bugs in Github issues that exercise Pelorus metrics
+- View the changes to the `Mean Time to Restore` and `Change Failure Rate` metrics.
 
-The Pelorus and todolist application can be installed automatically.  Using the forked copy of [mig-demo-apps](https://github.com/konveyor/mig-demo-apps) referenced as `https://github.com/<your_org>/mig-demo-apps`, execute the following steps:
+
+### Pelorus configuration
+
+>**Note:**
+> * A users [Github personal access token](https://github.com/settings/tokens) is required
+> * The `PROJECTS` key's value is the fork of the mig-apps-demo repository.
+
+* Update the /var/tmp/values.yaml to match the following text. You may already have the deploytime and committime exporters configured from a previous example.
```

Comment:
Update "exporters" section of the /var/tmp/values.yaml to match the following text.